### ISSUE TITLE
Pictures can't be opend

### ISSUE THREAD

**Issue by Lisiadito** [Lisiadito/removeInstagramLoginEnforcement#4]
Instagram must have recently changed something and even tough this plugin removes the login prompt it's not anymore possible to open pictures. (Only works with `right-click, open in new tab` 
I'm not sure if its something they have done with the css or if they removed an event listener.

**Comment by SuperSonicHub1** [comment]
Just came up with a fix. About to send a pull request. 
```javascript
document.body.addEventListener('click', e => {
    if (e.target.nodeName !== 'UL') {
      return
    }
    e.preventDefault();
    location.href = e.target.parentElement.parentElement.href;
})
```

**Comment by SuperSonicHub1** [comment]
#6

**Comment by Lisiadito** [comment]
Sweet thanks for the work! Will test it the next days and merge it when it works as expected.

**Comment by Lisiadito** [comment]
Fixed with #6
